src/warc2zim/language.py

```diff
@@ -0,0 +1,28 @@
+from zimscraperlib.i18n import get_language_details
+
+from warc2zim.constants import logger
+
+
+def parse_language(input_lang: str) -> str:
+    """Transform the input language into a valid ZIM Language Metadata
+
+    Support many ways to describe the language (all ISO 639 + English label)
+    Supports a comma-separated list of languages as input.
+    Deduplicates languages.
+    Ignore whitespaces.
+    Preserve language ordering (since it conveys meaning in ZIM metadata).
+    """
+
+    langs = []  # use a list to preserve order
+
+    for lang in [lang.strip() for lang in input_lang.split(",")]:
+        try:
+            lang_data = get_language_details(lang)
+            parsed_lang = lang_data["iso-639-3"]
+            if parsed_lang not in langs:
+                langs.append(parsed_lang)
+        except Exception:
+            logger.error(f"Invalid language setting `{lang}`. Using `eng`.")
+            return "eng"
```

Comment:
Looks like a `fra,unkown` would return `eng` while I would expect `fra`.
Shouldn't we log and skip un-parsable ones instead?

Reply:
You're right, this is silly.
Let's log and skip un-parsable ones.